src/tabs/smooth-scroll.ts

```diff
@@ -2,62 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 import { isMotionDisabled } from '@cloudscape-design/component-toolkit/internal';
 
-import isNativeSmoothScrollingSupported from './native-smooth-scroll-supported';
-
-interface ScrollContext {
-  scrollable: HTMLElement;
-  startX: number;
-  endX: number;
-  startTime: number;
-  scrollTime: number;
-}
-
-// The scroll speed depends on the scrolling distance. The equation below is an interpolation of measurements in Chrome.
-const getScrollSpeed = (pixels: number) => 0.0015 * Math.abs(pixels) + 0.558;
-const getScrollTime = (pixels: number) => Math.round(Math.abs(pixels) / getScrollSpeed(pixels));
-
-const now = () => (window.performance ? window.performance.now() : Date.now());
-
-const ease = (k: number): number => {
-  return 0.5 * (1 - Math.cos(Math.PI * k));
-};
-
-const step = (context: ScrollContext): void => {
-  const time = now();
-  const elapsed = Math.min((time - context.startTime) / context.scrollTime, 1);
-  const value = ease(elapsed);
-  const currentX = context.startX + (context.endX - context.startX) * value;
-  context.scrollable.scrollLeft = currentX;
-  // scroll more if we have not reached our destination
-  if (currentX !== context.endX) {
-    requestAnimationFrame(() => step(context));
-  }
-};
-
-const simulateSmoothScroll = (element: HTMLElement, endX: number): void => {
-  const startX = element.scrollLeft;
-  step({
-    scrollable: element,
-    startX,
-    endX,
-    startTime: now(),
-    scrollTime: getScrollTime(endX - startX),
-  });
-};
-
 const smoothScroll = (element: HTMLElement, to: number) => {
-  if (isMotionDisabled(element)) {
+  if (isMotionDisabled(element) || !element.scrollTo) {
     element.scrollLeft = to;
-    return;
-  }
-  if (isNativeSmoothScrollingSupported() && element.scrollTo) {
+  } else {
+    // istanbul ignore next: unit tests always have motion disabled
```

Comment:
I don't understand why this is not working properly. If I move the `ignore` comment above line 8, it still complains. I think we might just need to override the coverage check for this PR.

Reply:
(if this becomes a blocker, I'm happy to separate this out into a second PR. I just really need to deliver the scrolling update)

Reply:
Have you tried using `/* istanbul ignore else  */`? https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md#ignore-an-else-path

Reply:
I didn't, but according to the docs that's for cases where an `else` path might be required/expected but it doesn't exist at all.